atlasdb-api/src/main/java/com/palantir/atlasdb/spi/KeyValueServiceConfigHelper.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.spi;
+
+import java.util.Optional;
+
+public interface KeyValueServiceConfigHelper extends KeyValueServiceConfig {
+    @Override
+    String type();
+
+    @Override
+    default Optional<String> namespace() {
+        return Optional.empty();
```

Comment:
Yep. But this should be used in tests. It's mostly a hack to make it possible to initialize a KeyValueServiceConfig as
```
KeyValueServiceConfigHelper keyValueServiceConfig = () -> "Fake KVS";
```

I'd rather fail early with a NPE when accessing the namespace here, since it's not used anywhere. However, an improvement would be to make this class available only to test classes. Is there an easy way to do that in our project? I've tried adding it to the test project, without success.